### PR TITLE
Issue#SB-23758 DF>>Portal>>Design QA observations

### PR DESCRIPTION
**Pull request by ArtiSingh23**

### CODE REVIEW

File: projects/discussion-ui/src/lib/components/discussion-details/discussion-details.component.html [hunk 1]
```diff
@@ -47,42 +47,42 @@
...
-                <span class="pad-4" role="text"
+                <span class="ml-8" role="text"
                   [attr.aria-label]="'total upvoat is' + data?.upvotes">{{data?.downvotes}}</span>
               </span>
-              <span class="flex margin-right-l mobile" role="link" tabindex="0">
+              <span class="flex mr-24 mobile" role="link" tabindex="0">
                  <!-- trending -->
-                 <img src="./assets/discussion-ui/images/profits.png" alt="profit-image" width="14px" height="14px">
-                <span class="pad-4" i18n-aria-label aria-label="views count">{{data.viewcount}} views</span>
+                 <img src="./assets/discussion-ui/images/views.svg" alt="profit-image" width="16px" height="16px">
```
Comment (vinukumar-vs) [inline projects/discussion-ui/src/lib/components/discussion-details/discussion-details.component.html:82]: Why so many inline styles here for 
width & height of the image. 

The same code is duplicated for all the images. define the class & use the same class

File: projects/discussion-ui/src/lib/components/discussion-details/discussion-details.component.html [hunk 2]
```diff
@@ -21,18 +21,18 @@
       <ng-container *ngFor="let post of data?.posts">
         <div *ngIf="post?.index === 0">
         <!-- card content -->
-        <div class="margin-left-m mb-0-imp ">
+        <div class="post-reply-card">
           <!-- <div [innerHTML]="post.content | safeHtml"></div> -->
-          <p role="link" tabindex="0" class="discussion-heading fnormal mb-16 df-text-color" [innerHTML]="post?.content"></p>
-          <div class="discussion-content" role="link" tabindex="0">
+          <p role="link" tabindex="0" class="discussion-heading fnormal mb-8 df-text-color" [innerHTML]="post?.content"></p>
+          <div class="discussion-content mb-0" role="link" tabindex="0">
```
Comment (vinukumar-vs) [inline projects/discussion-ui/src/lib/components/discussion-details/discussion-details.component.html:27]: Remove the below commented code

File: projects/discussion-ui/src/lib/components/discussion-details/discussion-details.component.html [hunk 3]
```diff
@@ -126,11 +122,11 @@ <h3 class="fs-1 font-weight-bold"> Your answer here </h3>
         <div class="tabs-content">
           <div class="tabs-filter">
             <a href="javascript:void(0)" class="filter-option df-filter-option" role="link" [ngClass]="{'tabs-active' : currentFilter === 'timestamp',
-                            'ws-mat-accent-border':currentFilter !== 'timestamp'}" mat-button
+                            'ws-mat-accent-border':currentFilter !== 'timestamp'}"
```
Comment (vinukumar-vs) [inline projects/discussion-ui/src/lib/components/discussion-details/discussion-details.component.html:125]: Remove the below commented HTML code

File: projects/discussion-ui/src/lib/elements/category-card/category-card.component.html [hunk 4]
```diff
@@ -3,14 +3,12 @@
...
-      <div>
         <div class="category-card-header-label fs-1 font-weight-bold" [innerHTML]="category.name"></div>
-        <p class="category-card-header-text my-8 fnormal">{{category.description}}
-          <!-- <span class="dot">{{category.user.username}}</span> -->
-          <!-- <span class="margin-left-s">{{category.timestamp | date}}</span> -->
-        </p>
-      </div>
     </div>
+    <p class="category-card-header-text my-8 fnormal">{{category.description}}
+      <!-- <span class="dot">{{category.user.username}}</span> -->
```
Comment (vinukumar-vs) [inline projects/discussion-ui/src/lib/elements/category-card/category-card.component.html:9]: Remove the commented HTML Code

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: projects/discussion-ui/assets/styles/theme.scss [hunk 5]
```diff
@@ -40,28 +40,28 @@ html[layout='joy'] {
...
+  //   display: flex;
+  //   align-items: center;
+  //   cursor: pointer;
+  //   white-space: nowrap;
+  //   font-weight: bold;
+  //   transition: all ease 0.3s;
+  //   font-size: 0.75rem;
+  //   margin: 1rem 1.5rem 1rem 0;
+  //   height: 2.2rem;
+  // }
```
Comment (VenkatKadiveti) [inline projects/discussion-ui/assets/styles/theme.scss]: Remove commented code

File: projects/discussion-ui/src/lib/components/discuss-start/discuss-start.component.scss [hunk 6]
```diff
@@ -41,22 +41,20 @@
...
 					min-height: 2rem;
 					cursor: pointer;
 					outline:none;
+					padding: 0 0.875rem;
 					&:focus,.active{
-						border-color: #2196f3!important;
+						border-color: var(--primary-400) !important;
 					}
 				}
 			}
```
Comment (VenkatKadiveti) [inline projects/discussion-ui/src/lib/components/discuss-start/discuss-start.component.scss:60]: Maintain proper indentation both HTML, TS and SCSS

**Review by VenkatKadiveti — APPROVED** [review]

**Pull request merged**